src/main/CMV.java

```diff
@@ -94,7 +94,54 @@ public boolean lic5_calculate() {
         return false;
     }
 
-    private Boolean lic6_calculate() {
+    public Boolean lic6_calculate() {
```

Comment:
I am unsure wheter this is the correct solution to the assignment. From reading the description it sounds like a projection has to be made from the line onto each point (or the other way around) in order to get a appropiate distance.

Reply:
This is what I would argue that I'm doing.

1. The first for loop goes through each consecutive number of nodes and picks out the first and last points.
2. The second for loop goes through each point in between first and last, checking if the distance to the line is greater than `DIST`.

The distance equation is from: [Line defined by two points](https://en.wikipedia.org/wiki/Distance_from_a_point_to_a_line#Line_defined_by_two_points)